File: Array.java
import java.util.Scanner;
class Array{
public static void main(String args[])
{
int a[]=new int[4];
System.out.println("enter the no:");
 Scanner sc=new Scanner(System.in);
int s=sc.nextInt();

for(int i=0;i<s;i++)
{
a[i]=sc.nextInt();
System.out.println(a[i]);

}

System.out.println("array element are:"+a);

}
}

File: c++/Day2/array.cpp
#include<iostream>
using namespace std;
int main(){
	
	int a[] = {2,3,4,23,12,1,47};
	int max=*(a+0);
	
	for(int i=1; i<=6;i++){
		if(*(a+i)>max){
			max =*(a+i);
		}
			
	}	cout<<max;

int p[2][2];
int i,j;
cout<<"enter the element:";
for(int i=0;i<=1;i++)
{
	for(int j=0;j<=1;j++)
	{
		cin>>p[i][j];
	}
}
cout<<"The array element are :\n";
for(int i=0;i<=1;i++)
{
	for(int j=0;j<=1;j++)
	{
		cout<<p[i][j];
		cout<<"\t";
		
	}
	cout<<"\n";
}

}
